integreat_cms/static/src/js/dashboard/broken-links.ts

```diff
@@ -16,7 +16,20 @@ const getContent = async (url: string): Promise<Content> => {
     return response.json();
 };
 
+const showAllTotalNumbers = () => {
+    const elements = document.querySelectorAll<HTMLElement>(".total-results");
+
+    elements.forEach((element) => {
+        if (!element.closest("#broken-links")) {
+            const el = element;
+            el.classList.remove("hidden");
+        }
+    });
+};
+
```

Comment:
I can only find one element with the class total-results, iiuc we should be able to query one element and don't need to iterate over a list of elements.

Reply:
Hm, I'm not sure that works. All six rows should contain the class "total-results" for the string "(Insgesamt x)". I want to display five out of the six rows, just the one for broken-links not